lib/shared/link_preview_card.dart

```diff
@@ -83,28 +87,34 @@ class LinkPreviewCard extends StatelessWidget {
               ] else if (scrapeMissingPreviews)
                 SizedBox(
                   height: 150,
-                  child: hideNsfw
-                      ? ImageFiltered(
-                          imageFilter: ImageFilter.blur(sigmaX: 30, sigmaY: 30),
-                          child: LinkPreviewGenerator(
+                  // This is used for external links when Lemmy does not provide a preview thumbnail
+                  // and when the user has enabled external scraping.
+                  // This is only used in comfortable mode.
+                  child: Opacity(
```

Comment:
Is there any way to remove this usage of `Opacity`?

Reply:
I addressed this in my comment.

> - Link previews when Lemmy doesn't provide a thumbnail and scraping is on
>   - This is the only place I used `Opacity`, because I can't style the images within `LinkPreviewGenerator`. We already have this off by default with a warning that it hurts performance, so I hope this is ok. If not, I'll take it out!

So I think it's up to you whether this scenario (off by default + performance warning) is ok for `Opacity`, or if I should take it out, or if you have any other ideas to style `LinkPreviewGenerator` without forking. 😊

Reply:
Ahh I see, I missed that part! Funny that you mention forking `LinkPreviewGenerator`, because we are using a fork of it right now 😆

https://github.com/thunder-app/link_preview_generator

Reply:
> Funny that you mention forking `LinkPreviewGenerator`, because we are using a fork of it right now 😆

Aha! I remember @ajsosa was planning on doing that, but I missed that we already did it for one of @coslu's changes.

Alright, I have a PR open there: thunder-app/link_preview_generator#2. If/when that's approved and merged, I can update the package reference here, rip out `Opacity`, and we should be good to go!